apps/web/components/eventtype/EventTeamTab.tsx

```diff
@@ -394,10 +398,29 @@ export const EventTeamTab = ({
     );
   });
   const isManagedEventType = eventType.schedulingType === SchedulingType.MANAGED;
+  const [isOpenAssignmentWarnDialog, setIsOpenAssignmentWarnDialog] = useState<boolean>(false);
+  const [pendingRoute, setpendingRoute] = useState("");
+  const isConfirm = useRef(false);
```

Comment:
Can we rename `isConfirm` to something else - it's not obvious that it is used by the "AssignmentWarningDialog". How about `leaveWithoutAssigningHosts` ?